crates/solver/src/interactions/allowances.rs

```diff
@@ -408,14 +408,13 @@ mod tests {
                             serde_json::from_value::<CallRequest>(params[0].clone()).unwrap();
                         assert_eq!(call.data.unwrap(), allowance_call_data(owner, spender));
 
-                        match call.to.unwrap() {
-                            addr!("1111111111111111111111111111111111111111") => {
```

Comment:
Had to convert `mach` to `if` with equality checks because details of the `addr!` macro have changed which doesn't allow it to be used as a pattern anymore.